src/analytics.js

```diff
@@ -10,6 +10,10 @@
   // Skip analytics if preferred
   if (!w || w.HELP_IMPROVE_VIDEOJS === false) return;
 
+  if (n && n.doNotTrack && n.doNotTrack === '1') return;
+  if (n && n.msDoNotTrack && n.msDoNotTrack === '1') return;
+  if (w.doNotTrack && w.doNotTrack === '1') return;
```

Comment:
This looks good, but since we've otherwise commented this file pretty well, what do you think about adding a comment above this one too? Something like

```
// Respect Do Not Track in its various forms
```